packages/block-editor/src/components/use-block-sync/constants.js

```diff
@@ -0,0 +1,4 @@
+export const DIRECTION_IN = 1;
```

Comment:
`Non-blocking`
How do we feel about using string identifiers instead of numbers (Ex. 'DIRECTION_IN' instead of 1)? I think it would help with debugging.

<img width="875" alt="Screen Shot 2020-09-02 at 12 08 03 AM" src="https://user-images.githubusercontent.com/5414230/91942864-7afcab80-ecb0-11ea-9c87-4e67affbbaa9.png">

Reply:
I think this is fair! I originally switched from strings to numbers after making the constants because the constants are enough for code clarity, and I thought numbers would be less space. But a simple "in" "out" would probably be better